packages/rocketchat-ui/lib/chatMessages.coffee

```diff
@@ -114,14 +114,12 @@ class @ChatMessages
 		this.input.classList.add("editing")
 		this.$input.closest('.message-form').addClass('editing')
 
-		setTimeout =>
```

Comment:
@engelgabriel probably there was a reason, but now it is working without the timout